server.js

```diff
@@ -6,62 +6,47 @@ const cors = require('cors');
 const pg = require('pg');
 const jwt = require('jsonwebtoken');
 const passwordHash = require('password-hash');
+const config = require('./database_config.js');
+const authentication = require('./authentication.js');
 
 require('dotenv').config()
 
 const app = express();
-
 app.use(cors());
 app.use(bodyParser.json());
 
-const config = {
-  user: process.env.user,
-  database: process.env.database,
-  password: process.env.password, 
-  host: process.env.host, 
-  port: process.env.DATAPORT,
-  max: 10, 
-  idleTimeoutMillis: 30000,
-};
-
-const pool = new pg.Pool(config);
 const table = process.env.TABLE;
 const roomTable = process.env.ROOM_TABLE;
-
 const secretKey = process.env.KEY;
-let actualUserName;
-
-module.exports.query = function (text, values, callback) {
-  console.log('query:', text, values);
-  return pool.query(text, values, callback);
-};
 
+const querySettings = require('./database_query_settings.js');
 
+const pool = new pg.Pool(config);
+module.exports.query = querySettings;
 
-const authenticated = express.Router(); 
+app.use('/room', authentication);
 
-authenticated.use(function(req, res, next) {
-  let token = req.headers['auth'];
-  if (token) {
-    jwt.verify(token, secretKey, function(err, decoded) {      
-      if (err) {
-        return res.status(401).json({ "status": "error", "message": "Authentication required" });    
-      } else {
-        actualUserName = decoded.user;
-        next();
-      }
-    });
-  } else {
-    return res.status(401).json({ "status": "error", "message": "Authentication required" });
-  }
+app.post('/login', function(req,res){
+  loginPost(req,res);
 });
 
-app.use('/room', authenticated);
+app.post('/register', function(req, res){
```

Comment:
you can write `app.post('/register', registerPost)` 
it will call `registerPost` function with two parameters *(req and res)*